soc_091_global_peace_index/soc_091_global_peace_index_processing.py

```diff
@@ -0,0 +1,111 @@
+import urllib
+import os
+import pandas as pd
+from carto.datasets import DatasetManager
+from carto.auth import APIKeyAuthClient
+import boto3
+from botocore.exceptions import NoCredentialsError
+from zipfile import ZipFile
+
+
+# name of table on Carto where you want to upload data
+# this should be a table name that is not currently in use
+dataset_name = 'soc_091_global_peace_index' #check
+
+# first, set the directory that you are working in with the path variable
+# you can use an environmental variable, as we did, or directly enter the directory name as a string
+# example: path = '/home/soc_091_global_peace_index'
+path = os.getenv('PROCESSING_DIR')+dataset_name
+#move to this directory
+os.chdir(path)
+
+# create a new sub-directory within your specified dir called 'data'
+data_dir = 'data/'
+if not os.path.exists(data_dir):
+    os.mkdir(data_dir)
+
+'''
+Download data and save to your data directory
+'''
+# insert the url used to download the data from the source website
+url = 'http://visionofhumanity.org/app/uploads/2020/02/GPI-2019-overall-scores-2008-2019.xlsx' #check
+
+# download the data from source
+raw_data_file = data_dir+os.path.basename(url)
+urllib.request.urlretrieve(url, raw_data_file)
+
+'''
+Process data
+'''
+# read in csv file as Dataframe
+df=pd.read_excel(raw_data_file, sheet_name="Overall Scores", header=3) # selecting 2nd as row as the column names
+
+# Remove all unnecessary columns after column index 13
```

Comment:
```suggestion
# Remove all columns after column index 12 because they are empty
```

Reply:
Just noting here why they are unnecessary so a reader won't wonder if we are throwing out some of the data